eve/scripts/launch-end2end.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+# Helper script that check if the content and name of the branch is documentation related only.
+# This help us deciding whether we should or not launch the end2end tests.
+# Returns true when it's not documentation related and false when it is. 
+
+
+if git rev-parse --abbrev-ref HEAD | grep -q 'documentation/*' ; then
+    diff="$(git diff --name-only HEAD~1 | grep -v docs/)"
```

Comment:
might be valuable to move the `docs/` grep value to a variable as I feel there are more instances that we wouldn't need to run e2e ( or any tests? ). Such as the .github folder, res folder, readme.

Reply:
Good idea, gonna add that.